File: informatykaEuropejczyka/Rozdzial2/prog2_18.cpp
//wyznaczanie maksimum w n elementowym ciagu liczb
#include <iostream>
using namespace std;
const int MAX = 100;

int oblicz (int T[], int n)
{
 int maksimum=T[0];
 for (int i=1;i<n;i++)
  if (T[i]>maksimum) maksimum=T[i];
 return maksimum;
}

int main()
{
 int T[MAX], n;
 cout<<"podaj liczbe elementow tablicy: ";
 cin>>n;
 cout<<"podaj elementy tablicy:"<<endl;
 for (int i=0;i<n;i++)
 {
  cout<<"T["<<i<<"] = ";
  cin>>T[i];
 }
 cout<<"maksimum = "<<oblicz(T,n)<<endl;
 return 0;
}

File: informatykaEuropejczyka/Rozdzial2/prog2_18.py
#wyznaczanie maksimum w n elementowym ciagu liczb

def oblicz (T):
    n=len(T)
    maksimum=T[0]
    for i in range(1,n):
        if T[i]>maksimum:
            maksimum=T[i]
    return maksimum    

#print(oblicz([5,3,4,2,8,7,6,1,0,9]))

